tools/trap/ramdumpParser.py

```diff
@@ -324,8 +323,8 @@ def read_address(self, addr, length, debug=False):
 			a = self.elf_file_fd.read(length)
 		return a
 
-	# returns a tuple of the result by reading address from the "specified format string"
-	# return None on failure
+	# Returns a tuple of the result by reading address from the "specified format string"
+	# Return None on failure
```

Comment:
There are `Returns` and `Return`.
Could you check which is correct?

Reply:
I think each usage is correct. Nonetheless, I have rephrased the comment.